zebra/explorer.md

```diff
@@ -13,23 +13,23 @@ https://zcashblockexplorer.com/
-| getblockchaininfo           | YES, partially.    | In the explorer this information is appended to an `info` array.                                                                                                                                                    | [Zcash](https://zcash.github.io/rpc/getblockchaininfo.html) - [Zebra](https://doc.zebra.zfnd.org/zebra_rpc/methods/trait.Rpc.html#tymethod.get_blockchain_info)              | [1](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer/metrics/metrics_warmer.ex#L14)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| getblockhashes              | NO.                | As it is used in different places, with different arguments, it need all the fields and options.<br>(noOrphans is always true.)                                                                                                                    | [Zcash](https://zcash.github.io/rpc/getblockhashes.html)                                                                                                                     | [1](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer/blocks/block_warmer.ex#L19) - [2](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer/transactions/transaction_cache.ex#L19) - [3](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/block_controller.ex#L54) - [4](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/block_controller.ex#L89)                                                                                                                              |
```

Comment:
Added: (noOrphans is always true.)

Seems like the diff isn't working on GitHub, probably due to the whitespace.